src/main/java/com/getbase/http/HttpClient.java

```diff
@@ -86,6 +86,10 @@ public Response request(HttpMethod method,
         return response;
     }
 
+    protected Request.Builder applyAuthorization(Request.Builder requestBuilder) {
```

Comment:
As we don't mention this feature in README we could at least add a comment here, like "Override this method if you want to use custom authorization method".